lib/datadog/appsec/ext.rb

```diff
@@ -5,6 +5,10 @@ module AppSec
     module Ext
       INTERRUPT = :datadog_appsec_interrupt
       SCOPE_KEY = 'datadog.appsec.scope'
+
+      TAG_APPSEC_ENABLED = '_dd.appsec.enabled'
+      TAG_APM_ENABLED = '_dd.apm.enabled'
```

Comment:
does this constant really belong to `Appsec` module?

Reply:
I believe so as the only time it is added to a trace is when ASM standalone is activated. Usually, we don't want disable APM, this is for now the only case when it is disabled.